engine/particle/src/wscript

```diff
@@ -34,7 +34,7 @@ def build(bld):
                   includes = ['.', '../proto'],
                   target = 'particle_shared',
                   protoc_includes = '../proto',
-                  use = 'DDF_NOASAN DLIB_NOASAN SOCKET PROFILE_NULL_NOASAN GRAPHICS_PROTO_NOASAN',
+                  use = 'DDF_NOASAN DLIB_NOASAN SOCKET PROFILE_NULL_NOASAN PLATFORM_NULL GRAPHICS_NOASAN',
```

Comment:
Just a reminder to test with `--with-asan` to make sure you don't need a `PLATFORM_NULL_NOASAN`